src/components/VulnerabilitiesBlock/CcviThermometer/CcviThermometer.stories.tsx

```diff
@@ -7,17 +7,17 @@ export default {
 };
 
 export const Score00 = () => {
-  return <CcviThermometer overallScore={0} />;
+  return <CcviThermometer overallScore={0} regionName="region name" />;
 };
 
 export const Score04 = () => {
-  return <CcviThermometer overallScore={0.4} />;
+  return <CcviThermometer overallScore={0.4} regionName="region name" />;
 };
 
 export const Score09 = () => {
-  return <CcviThermometer overallScore={0.9} />;
+  return <CcviThermometer overallScore={0.9} regionName="region name" />;
 };
 
 export const Score10 = () => {
-  return <CcviThermometer overallScore={1.0} />;
+  return <CcviThermometer overallScore={1.0} regionName="region name" />;
```

Comment:
Receiving the entire title might make the component a bit more reusable and make it more flexible if we need to change the copy based on other factors

Reply:
punting for now but adding to the list